src/components/AFEpage/index.js

```diff
@@ -1,85 +1,199 @@
-            Started in 2016 by IIT-D Alumnus, the organization equips the youth
-            with 21st-century skills through programming, coding, critical
-            thinking, and problem-solving with a commitment to support its
-            students until their placements.
-          </p>
-          <p>
-            Navgurukul offers a 1-year fully-funded skilling and recruitment
-            program in software engineering that enables them to secure
-            aspirational careers.
-          </p>
-        </div>
-        <div className="AFE-details">
-          <h2 className="heading">Our Approach</h2>
-          <p>
-            Navgurukul is working with a two-step approach to make our students
-            CS ready and placed in tech jobs.
-            <ul>
-              <li>
-                <div className="AFE-approaches-title">
-                  Accessibility and readiness program: Meraki
-                </div>
-                Age group: 13+ girls/ women
-              </li>
-              <li>
-                <div className="AFE-approaches-title">
-                  Skilling and job placement program: Residential Program
-                </div>
-                Age group 18 - 29 years
-              </li>
-            </ul>
-          </p>
-        </div>
-        <div className="AFE-details">
-          <h2 className="heading">NavGurukul 🤝 Amazon Future Engineer</h2>
-          <p>
-            Amazon Future Engineer (AFE) is a comprehensive childhood-to-career
-            program to increase access to computer science education for
-            children and young adults who typically lack these opportunities.
-          </p>
-          <p>
-            To take its mission forward in India, AFE has partnered with
-            Navguruku in its Meraki program to close the gender gap in
-            technology. Meraki is an android application focused on making
-            programming and digital skills accessible to learners from different
-            communities through a smartphone to create direct job opportunities
-            or admissions to aspirational training programs. The focus area is
-            young women and girls to bridge the gender gap in the digital
-            literacy sector.
-          </p>
-        </div>
-      </div>
-    </div>
+    <>
+      <Container maxWidth="lg">
+        <Grid className={classes.Grid_Space}>
+          <Grid>
+            <img
+              src={require("./asset/amazon.png")}
+              alt="Closing The Gender Gap in Technology"
+            />
+          </Grid>
+          <Grid>
+            <img
+              src={require("./asset/futurengineer.png")}
+              alt="Closing The Gender Gap in Technology"
+            />
+          </Grid>
+        </Grid>
+        <Grid
+          container
+          spacing={!isActive ? 3 : 4}
+          className={classes.Grid_Space}
+        >
+          <Grid item xs={12} sm={6} md={6}>
```

Comment:
As noted at https://mui.com/material-ui/api/grid/ :

> The value is applied for the sm breakpoint and wider screens if not overridden

So, if the number of columns for `sm` and `md` are the same, the `md` is not needed (also lines 48, 79, 95, 109, 116):

```suggestion
          <Grid item xs={12} sm={6}>
```